### PR TITLE
Add telemetry data about metrics in, metrics scraping and trace in metrics to zipkin receiver.

### PR DESCRIPTION
**Pull request by Ax1an**
#6498 

I find that telemetry data about log in metrics have been added. So I added telemetry data about metrics in, metrics scraping and trace in metrics to zipkin receiver.

- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/server-fetcher-plugin/prometheus-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/fetcher/prometheus/provider/PrometheusFetcherProvider.java [hunk 1]
```diff
@@ -62,8 +67,22 @@
...
+
+    private final CounterMetrics errorCounter;
+
     public PrometheusFetcherProvider() {
         config = new PrometheusFetcherConfig();
+        MetricsCreator metricsCreator = getManager().find(TelemetryModule.NAME)
+                .provider()
+                .getService(MetricsCreator.class);
+        histogram = metricsCreator.createHistogramMetric(
+                "metrics_scratch_latency", "The process latency of metrics scratching",
```
Comment (wu-sheng) [inline oap-server/server-fetcher-plugin/prometheus-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/fetcher/prometheus/provider/PrometheusFetcherProvider.java:80]:
```suggestion
                "metrics_fetcher_latency", "The process latency of metrics scratching",
```
`scratch` -> `fetcher`.

**Comment by wu-sheng** [comment]
I think somehow you break tests?

**Comment by Ax1an** [comment]
@wu-sheng I probably know what the problem is and I'll try to fix it.

**Review by kezhenxu94 — COMMENTED** [review]
Seems `scratch` / `scratching` is a typo of `scrap` / `scraping`, please correct them in both title, commit message, codes, comments, etc.

**Comment by Ax1an** [comment]
@kezhenxu94 Done.

**Review by kezhenxu94 — DISMISSED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**